scripts/skills/perks/perk_rf_swift_stabs.nut

```diff
@@ -1,5 +1,9 @@
 this.perk_rf_swift_stabs <- ::inherit("scripts/skills/skill", {
 	m = {
+		RequiredWeaponType = ::Const.Items.WeaponType.Dagger,
+		RequiredDamageType = ::Const.Damage.DamageType.Piercing,
+		ActionPointCostModifier = -2,
+		ActionPointCostMin = 2,
 		IsSpent = true
```

Comment:
I would like a short explanation and/or renaming of this variable. 
Usually `IsSpent` is used for a perk that is only used once per turn.
Here it seems to be used to control whether this perk is in effect or not, which confuses me when trying to understand this code. And you seem to spend and un-spend this perk multiple times per turn

Reply:
I'll change it to `IsInEffect`